apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/MetadataInfoTabs/index.tsx

```diff
@@ -42,10 +42,8 @@ export const MetadataInfoTabs = forwardRef<HTMLDivElement, Props>(
             </div>
           ) : null}
         </div>
-        <div className='w-full custom-scrollbar overflow-y-auto'>
-          <div className='flex px-4 py-5 flex-col gap-4 w-full overflow-x-auto'>
-            {children({ selectedTab })}
-          </div>
+        <div className='w-full flex px-4 pb-5 mt-5 flex-col gap-4 custom-scrollbar overflow-auto relative'>
```

Comment:
Try to generate horizontal scroll just in case to see how it looks

Reply:
Messages are wrapped, so it does seem it is not able to overflow horizontally